contracts/contracts/strategies/NativeStaking/ValidatorAccountant.sol

```diff
@@ -0,0 +1,258 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { ValidatorRegistrator } from "./ValidatorRegistrator.sol";
+import { IWETH9 } from "../../interfaces/IWETH9.sol";
+
+/// @title Validator Accountant
+/// @notice Attributes the ETH swept from beacon chain validators to this strategy contract
+/// as either full or partial withdrawals. Partial withdrawals being consensus rewards.
+/// Full withdrawals are from exited validators.
+/// @author Origin Protocol Inc
+abstract contract ValidatorAccountant is ValidatorRegistrator {
+    /// @notice The minimum amount of blocks that need to pass between two calls to manuallyFixAccounting
+    uint256 public constant MIN_FIX_ACCOUNTING_CADENCE = 7200; // 1 day
+
+    /// @notice Keeps track of the total consensus rewards swept from the beacon chain
+    uint256 public consensusRewards;
+
+    /// @notice start of fuse interval
+    uint256 public fuseIntervalStart;
+    /// @notice end of fuse interval
+    uint256 public fuseIntervalEnd;
+    /// @notice last block number manuallyFixAccounting has been called
+    uint256 public lastFixAccountingBlockNumber;
+
+    uint256[49] private __gap;
```

Comment:
I think the usual best practice is to make the initial size of the gap variable such that the total storage slots used fit into some nice round number. If a contract inheritance layer had one storage slot used, it would make sense to gap 49, which we do elsewhere. But here with 4 storage slots used, a gap of 46 might make more sense.

Reply:
the problem is we have been deploying to Holesky whilst making changes. That's why its now 49 and not 50 which it started at.

Reply:
If we already have contracts live that we are upgrading, it is what is. We'll keep.